<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>To1BitDialog</class>
 <widget class="QDialog" name="To1BitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2値化</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>2値化法</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>しきい値法(&amp;B)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>しきい値(&amp;T):</string>
           </property>
           <property name="buddy">
            <cstring>spinBoxT</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSlider" name="sliderT">
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="maximum">
              <number>256</number>
             </property>
             <property name="value">
              <number>128</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxT">
             <property name="maximum">
              <number>256</number>
             </property>
             <property name="value">
              <number>128</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonE">
        <property name="text">
         <string>誤差拡散法(&amp;E)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonB</tabstop>
  <tabstop>sliderT</tabstop>
  <tabstop>spinBoxT</tabstop>
  <tabstop>radioButtonE</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>To1BitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>To1BitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonB</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderT</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxT</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>thresholdChanged(int)</signal>
  <slot>setThreshold(int)</slot>
 </slots>
</ui>
